src/main/java/in/zapr/druid/druidry/aggregator/StringAggregator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-present Red Brick Lane Marketing Solutions Pvt. Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package in.zapr.druid.druidry.aggregator;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+abstract class StringAggregator extends DruidAggregator {
```

Comment:
No need for this this abstraction as all other `Aggregator` like `Long`, `Double`, etc in this library are direct extension of `DruidAggregator`, we could do same thing here.